core/src/test/java/org/springframework/security/authorization/AuthorizationAdvisorProxyFactoryTests.java

```diff
@@ -335,7 +335,7 @@ public void setTargetVisitorThenUses() {
 	@Test
 	public void setTargetVisitorIgnoreValueTypesThenIgnores() {
 		AuthorizationAdvisorProxyFactory factory = AuthorizationAdvisorProxyFactory.withDefaults();
-		assertThatExceptionOfType(ClassCastException.class).isThrownBy(() -> ((Integer) factory.proxy(35)).intValue());
+		assertThatExceptionOfType(ClassCastException.class).isThrownBy(() -> (factory.proxy(35)).intValue());
```

Comment:
It looks like the parentheses are no longer needed around `factory.proxy(35)`. Can you remove them please?